src/utf8.cc

```diff
@@ -23,7 +23,7 @@ namespace wabt {
 namespace {
 
 const int s_utf8_length[256] = {
- // 0  1  2  3  4  5  6  7  8  9  a  b  c  d  e  f
+    // 0  1  2  3  4  5  6  7  8  9  a  b  c  d  e  f
     1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,  // 0x00
```

Comment:
This is worse since the columns no longer line up

Reply:
Added `// clang-format off` and `// clang-format on` around this array.